tools.json

```diff
@@ -12757,6 +12757,47 @@
         "RUST",
         "SCALA"
       ]
+    },
+    {
+      "name": "Hermeto",
+      "publisher": "The Hermeto Project",
+      "description": "Hermeto is a CLI tool that pre-fetches your project's dependencies, which enables network-isolated builds, which enables the production of high-accuracy SBOMs",
+      "repository_url": "https://github.com/hermetoproject/hermeto",
+      "website_url": "https://hermetoproject.github.io/hermeto/",
+      "capabilities": [
+        "SBOM"
+      ],
+      "availability": [
+        "OPEN_SOURCE",
+        "OSI_APPROVED"
+      ],
+      "functions": [
+        "AUTHOR",
+        "PACKAGE_MANAGER_INTEGRATION"
```

Comment:
I think you're missing `TRANSFORM` here, we support SPDX as well and the way we currently do that is by transforming our base abstract SBOM representation (which conforms to the CycloneDX standard) to SPDX primitives. This is currently a one way transformation only, but we already know we'll probably have to support both natively which will only emphasize the `TRANSFORM` capability more.

Reply:
https://github.com/ossf/sbom-everywhere/pull/121#discussion_r2182946122. Same reasoning here, please disregard `TRANSFORM`.